README.md

```diff
@@ -1,33 +1,50 @@
-| com.avaloq.tools.ddk.xtext.ui          | Eclipse UI part of runtime library extensions            |
-| com.avaloq.tools.ddk.xtext.builder     | Extensions to Xtext builder                              |
-| com.avaloq.tools.ddk.xtext.format.\*   | Format DSL                                               |
-| com.avaloq.tools.ddk.xtext.check.\*    | Check DSL                                                |
-| com.avaloq.tools.ddk.xtext.scope.\*    | Scope DSL                                                |
-| com.avaloq.tools.ddk.xtext.export.\*   | Export DSL                                               |
-| com.avaloq.tools.ddk.typesystem        | Basics for building a typesystem for a DSL               |
+| Plugin                                 | Description                                                |
+|----------------------------------------|------------------------------------------------------------|
+| com.avaloq.tools.ddk.workflow          | Mwe2 workflows to regenerate DDK languages                 |
+| com.avaloq.tools.ddk.xtext.generator   | Mwe2 workflow fragements for generating DSL implementation |
+| com.avaloq.tools.ddk.xtext             | Extensions to Xtext runtime library                        |
+| com.avaloq.tools.ddk.xtext.common.\*   | Optional extensions to Xtext runtime library               |
+| com.avaloq.tools.ddk.xtext.ui          | Eclipse UI part of runtime library extensions              |
+| com.avaloq.tools.ddk.xtext.builder     | Extensions to Xtext builder                                |
+| com.avaloq.tools.ddk.xtext.format.\*   | Format DSL                                                 |
+| com.avaloq.tools.ddk.xtext.check.\*    | Check DSL                                                  |
+| com.avaloq.tools.ddk.xtext.checkcfg.\* | Check Configuration DSL                                    |
+| com.avaloq.tools.ddk.xtext.scope.\*    | Scope DSL                                                  |
+| com.avaloq.tools.ddk.xtext.export.\*   | Export DSL                                                 |
+| com.avaloq.tools.ddk.xtext.valid.\*    | obsolete and is replaced by Check DSL                      |
+| com.avaloq.tools.ddk.typesystem        | Basics for building a typesystem for a DSL                 |
+| com.avaloq.tools.ddk.test.\*           | Support for writing tests for DSLs                         |
+| com.avaloq.tools.ddk.xtextspy.\*       | View to support debugging a DSL                            |
 
-
-#Overview
+| Feature                                | Description                                                          |
+|----------------------------------------|----------------------------------------------------------------------|
+| com.avaloq.tools.ddk.runtime.feature   | Runtime extension to Xtext                                           |
+| com.avaloq.tools.ddk.feature           | Toolkit to install into Eclipse on top of Xtext for DSL development  |
 
 
+# Overview
```

Comment:
Being in overview and short description of the project this section should be first before line 7.